07-routing-gossip.md

```diff
@@ -636,6 +690,34 @@ timeouts.  It also causes a natural ratelimiting of queries.
     * [`1`:`complete`]
     * [`2`:`len`]
     * [`len`:`encoded_short_ids`]
+    * [`query_channel_range_tlv`]
```

Comment:
the name `query_channel_range_tlv` confused me a bit. might it make more sense to call it `reply_channel_range_tlv`? there is a reference further down that would also need to be updated. correct if i'm misunderstanding the rationale :)

Reply:
You're right, I will rename this field